rules/phasing.smk

```diff
@@ -0,0 +1,59 @@
+HAPCUT2=config["hapcut2"]
+
+rule HAPCUT2_extractHAIRS:
```

Comment:
I'm not sure if I remember correctly but I believe rules were functions and therefore should be written with underline separation according to [pep8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).
```suggestion
rule hapcut2_extract_hairs:
```

Reply:
And I see HapCUT2 doesn't follow these conventions (it's also not written in python) but I think we should follow pep8 to the best of out ability whenever possibility.

Reply:
I am not sure I would agree that the naming of rules should **have to** follow python convention. These are not python objects as they are specific to snakemake. But I also don't care much either way. If you feel this would improve consistency we could set this as a practise for this project.

Reply:
I vaguely would like @marcelm mentioning something about snakemake rules being functions and since I can't find it now, he would have to remind me whether my memory serves me right or not.

Either way, looking at the [snakemake docs](https://snakemake.readthedocs.io/en/stable/) every example name is written as lower_case functions in python (assuming NAME is just written as a placeholder) so I'd prefer to follow that if there are no established conventions.

Reply:
They are functions under the hood, but if that weren’t the case, they kind of look like functions, so making them consistently lowercase would make sense.